app/models/staypuft/deployment/cinder_service.rb

```diff
@@ -202,14 +202,23 @@ def lvm_ptable
       end
     end
 
-    %w{hostname login password server_port storage_familie transport_type storage_protocol
-        nfs_share nfs_shares_config volume_list vfiler vserver controller_ip sa_password 
-        storage_pool }.each do |name|
+    %w{hostname login password server_port transport_type storage_protocol storage_family
```

Comment:
this results in compute_netapp_storage_familys not compute_netapp_storage_families